classes/profile_table.php

```diff
@@ -108,15 +108,25 @@ public function col_scripttype(object $record): string {
      */
     public function col_request(object $record): string {
         $displayedrequest = $record->request . $record->pathinfo;
-        if ($record->method === 'GET' && !empty($record->parameters)) {
-            $displayedrequest .= '?' . urldecode($record->parameters);
+        if (!empty($record->parameters)) {
+            if ($record->scripttype == profile::SCRIPTTYPE_CLI) {
+                // For CLI scripts, request should look like `command.php --flag=value` as an example.
+                $separator = ' ';
```

Comment:
we can escape this with escapeshellcmd so you can cut and paste it into the cli and have it still work.

Reply:
![image](https://user-images.githubusercontent.com/9924643/146695444-208d9943-98c5-4ba9-b731-9543d43f0852.png)

Output